src/de/schildbach/pte/util/HttpClient.java

```diff
@@ -322,10 +330,7 @@ public static HttpUrl testRedirect(final HttpUrl base, final String content) {
     public static boolean testExpired(final String content) {
         // check for expired session
         final Matcher mSessionExpired = P_EXPIRED.matcher(content);
-        if (mSessionExpired.find())
-            return true;
-
-        return false;
+        return mSessionExpired.find();
```

Comment:
This is an unrelated change. I'd prefer to put such changes into a different commit/PR to keep this PR as small as possible,